bot_template.py

```diff
@@ -1,20 +1,19 @@
-from irc import *
-from secrets import *
-import os
-import random
-import time
+from irc import IRC
+from secrets import get_channel, get_server
+
 
 channel = get_channel()
 server = get_server()
 nickname = "test_bot"
 
-irc = IRC()
-irc.connect(server, channel, nickname)
+
+irc_obj = IRC()
```

Comment:
I would say lets keep the name of the instance to "irc" for the sake of consistency, simplicity, and syntactically more readable for function calls. Consistency in that the instance creation of the irc object in the class is set to "irc". Syntactically readable as in when calling a class method, `irc.send()` or `irc.connect()` reads better than `irc_obj.send()` or `irc.connect()` and so on.

Reply:
I personally like to keep it easy on the eyes, since your import module is named "irc.py" and so is your instance, I thought naming the instance "irc_obj" would be easy on the eyes, but since this is your repository, I'll stick with the recommended instance name.